charts/nextcloud/files/nginx.config.tpl

```diff
@@ -3,7 +3,17 @@ upstream php-handler {
 }
 
 server {
+    {{- if and (has "IPv4" .Values.nginx.ipFamilies) (has "IPv6" .Values.nginx.ipFamilies) }}
+    # Both IPv4 and IPv6 are enabled
+    listen [::]:{{ .Values.nginx.containerPort }};
+    ipv6only = off;
+    {{- else if and (not (has "IPv4" .Values.nginx.ipFamilies)) (has "IPv6" .Values.nginx.ipFamilies) }}
+    # Only IPv6 is enabled
+    listen [::]:{{ .Values.nginx.containerPort }};
+    {{- else }}
+    # Default, IPv4 only
     listen {{ .Values.nginx.containerPort }};
+    {{- end }}
```

Comment:
```suggestion
    {{- if has "IPv6" .Values.nginx.ipFamilies }}
    listen [::]:{{ .Values.nginx.containerPort }};
    {{- if has "IPv4" .Values.nginx.ipFamilies }}
    ipv6only = off;
    {{- end }}
    {{- else }}
    # Default, IPv4 only
    listen {{ .Values.nginx.containerPort }};
    {{- end }}
```
I think this logic is a bit simpler

Reply:
Absolutely!

Reply:
I will need to revert this, the `ipv6only=off` setting needs to be on the same line as the listen.